aioesphomeapi/api.proto

```diff
@@ -1161,7 +1161,7 @@ message BluetoothLEAdvertisementResponse {
   option (no_delay) = true;
 
   uint64 address = 1;
-  string name = 2;
+  bytes name = 2;
```

Comment:
Changed to bytes so we can decode it ourselves with `replace` since we cannot change the source of the bad data but we can prevent it from causing a failure or the device from not working